unison-src/transcripts/idempotent/dependents-dependencies-debugfile.md

```diff
@@ -92,17 +92,16 @@ scratch/main> dependencies d
 
     Types:
 
-    1. Boolean
-    2. Nat
```

Comment:
This seems to have fixed a small bug in `dependencies`: this term doesn't depend directly on `Nat`.

Reply:
We must have been counting literals as a reference to the type but i guess that's inconsistent with how we treat other nonliteral values